Update contributor's guide (#502)
cadet/CADET-Core: CONTRIBUTING.md

## Contributing

CADET-Core is an open-source project and we are very happy to accept contributions from the community.
Please feel free to open issues or submit patches (preferably as pull requests) at any time.

> For planned larger contributions, it might be beneficial to get in touch with us first, e.g. through the [CADET forum](https://forum.cadet-web.de/).

Reviewer: ```suggestion
For planned larger contributions, we would encourage you to get in touch with us first, e.g. through the [CADET forum](https://forum.cadet-web.de/).
```
